### PR TITLE
chore: Update Auth docs for Amplify Flutter v1

### PR DESCRIPTION
**Pull request by Jordan-Nelson**
#### Description of changes:
- Update Flutter Auth docs to use top level version switcher
- Update outdated examples

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [x] iOS
- [x] Android
- [x] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [x] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [x] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/fragments/lib/auth/ios/signin/20_confirmSignUp.mdx [hunk 1]
```diff
@@ -43,3 +43,9 @@ func confirmSignUp(for username: String, with confirmationCode: String) -> AnyCa
 </Block>
 
 </BlockSwitcher>
+
+You will know the sign up flow is complete if you see the following in your console window:
```
Comment (Jordan-Nelson) [inline src/fragments/lib/auth/ios/signin/20_confirmSignUp.mdx:47]: Note: This language was preexisting. It was moved from the shared fragments to allow flutter to omit this.

File: src/fragments/lib/auth/ios/signin/30_signIn.mdx [hunk 2]
```diff
@@ -47,3 +47,9 @@ func signIn(username: String, password: String) -> AnyCancellable {
 </Block>
 
 </BlockSwitcher>
+
+You will know the sign in flow is complete if you see the following in your console window:
```
Comment (Jordan-Nelson) [inline src/fragments/lib/auth/ios/signin/30_signIn.mdx:51]: Note: This language was preexisting. It was moved from the shared fragments to allow flutter to omit this.

**Review by dnys1 — COMMENTED** [review]
Still working through. Some initial thoughts.

File: src/fragments/lib/auth/flutter/device_features/30_fetchDevice.mdx [hunk 3]
```diff
@@ -3,10 +3,10 @@ Future<void> fetchAllDevices() async {
   try {
     final devices = await Amplify.Auth.fetchDevices();
     for (final device in devices) {
-        print('Device: $device');
+        safePrint('Device: $device');
```
Comment (dnys1) [inline src/fragments/lib/auth/flutter/device_features/30_fetchDevice.mdx:6]:
```suggestion
      safePrint('Device: $device');
```

File: src/fragments/lib/auth/flutter/access_credentials/10_fetchAuthSession.mdx [hunk 4]
```diff
@@ -1,78 +1,33 @@
...
   try {
-    final result = await Amplify.Auth.fetchAuthSession().timeout(
-      const Duration(seconds: 5),
-    );
-    final identityId = (result as CognitoAuthSession).identityId!;
+    final cognitoPlugin = Amplify.Auth.getPlugin(AmplifyAuthCognito.pluginKey);
+    final result = await cognitoPlugin.fetchAuthSession().timeout(
+          const Duration(seconds: 5),
+        );
+    final identityId = (result).identityIdResult.value;
```
Comment (dnys1) [inline src/fragments/lib/auth/flutter/access_credentials/10_fetchAuthSession.mdx:27]:
```suggestion
    final identityId = result.identityIdResult.value;
```

File: src/fragments/lib/auth/flutter/access_credentials/10_fetchAuthSession.mdx [hunk 5]
```diff
@@ -1,78 +1,33 @@
...
     safePrint('identityId: $identityId');
   } on Exception catch (e) {
```
Comment (dnys1) [inline src/fragments/lib/auth/flutter/access_credentials/10_fetchAuthSession.mdx:29]:
```suggestion
  } on AuthException catch (e) {
```

File: src/fragments/lib/auth/flutter/delete_user/10_delete_user.mdx [hunk 6]
```diff
@@ -2,9 +2,9 @@
 Future<void> deleteUser() async {
   try {
     await Amplify.Auth.deleteUser();
-    print('Delete user succeeded');
+    safePrint('Delete user succeeded');
   } on Exception catch (e) {
```
Comment (dnys1) [inline src/fragments/lib/auth/flutter/delete_user/10_delete_user.mdx:6]:
```suggestion
  } on AuthException catch (e) {
```

File: src/fragments/lib/auth/flutter/device_features/10_rememberDevice.mdx [hunk 7]
```diff
@@ -2,9 +2,9 @@
 Future<void> rememberCurrentDevice() async {
   try {
     await Amplify.Auth.rememberDevice();
-    print('Remember device succeeded');
+    safePrint('Remember device succeeded');
   } on Exception catch (e) {
```
Comment (dnys1) [inline src/fragments/lib/auth/flutter/device_features/10_rememberDevice.mdx:6]:
```suggestion
  } on AuthException catch (e) {
```

File: src/fragments/lib/auth/flutter/device_features/20_forgetDevice.mdx [hunk 8]
```diff
@@ -6,9 +6,9 @@
 Future<void> forgetCurrentDevice() async {
   try {
     await Amplify.Auth.forgetDevice();
-    print('Forget device succeeded');
+    safePrint('Forget device succeeded');
   } on Exception catch (e) {
```
Comment (dnys1) [inline src/fragments/lib/auth/flutter/device_features/20_forgetDevice.mdx:10]:
```suggestion
  } on AuthException catch (e) {
```

File: src/fragments/lib/auth/flutter/device_features/20_forgetDevice.mdx [hunk 9]
```diff
@@ -17,13 +17,15 @@ Future<void> forgetCurrentDevice() async {
...
+AuthDevice myDevice;
+
 Future<void> forgetSpecificDevice() async {
   try {
-    const device = CognitoDevice(id: 'us-west-2_38284cea-6c7f-4a8c-bcfa-ac8946a0d1eb');
-    await Amplify.Auth.forgetDevice(device);
-    print('Forget device succeeded');
+    await Amplify.Auth.forgetDevice(myDevice);
+    safePrint('Forget device succeeded');
   } on Exception catch (e) {
```
Comment (dnys1) [inline src/fragments/lib/auth/flutter/device_features/20_forgetDevice.mdx:27]:
```suggestion
  } on AuthException catch (e) {
```

File: src/fragments/lib/auth/flutter/device_features/30_fetchDevice.mdx [hunk 10]
```diff
@@ -3,10 +3,10 @@ Future<void> fetchAllDevices() async {
...
     }
   } on Exception catch (e) {
```
Comment (dnys1) [inline src/fragments/lib/auth/flutter/device_features/30_fetchDevice.mdx:8]:
```suggestion
  } on AuthException catch (e) {
```

**Review by dnys1 — APPROVED** [review]
LGTM

**Review by thisisabhash — APPROVED** [review]

**Pull request merged**